modules/300-prometheus/openapi/config-values.yaml

```diff
@@ -17,6 +17,8 @@ properties:
 
       `storageClass: false` — forces the `emptyDir` usage. You will need to delete the old PVC and restart the Pod manually.
 
+      **CAUTION!** When migrating Prometheus with the specified storage Class name, the pods will hang in the Pending state and it will be necessary to delete the old PVC and restart the pod manually. The transfer of Prometheus to other nodes when using emptyDir takes place without the need to restart the hearths.
```

Comment:
```suggestion
      **CAUTION!** When migrating Prometheus with the specified storageClass name, the pods will hang in Pending state and it will be necessary to delete the old PVC and manually restart the pod. Migration of Prometheus to other nodes using emptyDir occurs without the need to restart the pods.
```